linera-client/src/benchmark.rs

```diff
@@ -601,6 +628,47 @@ impl<Env: Environment> Benchmark<Env> {
         Ok(())
     }
 
+    fn create_operation(
+        fungible_application_id: Option<ApplicationId>,
+        recipient_chain_id: ChainId,
+        owner: AccountOwner,
+        amount: Amount,
+    ) -> Operation {
+        match fungible_application_id {
+            Some(application_id) => {
+                Self::fungible_transfer(application_id, recipient_chain_id, owner, owner, amount)
```

Comment:
The recipient seems to be the same account owner as the sender. This may create large ledgers (map views) and slow down the chains. Also, the received money won't be usable by the owner of the receiver chain. I don't have an easy fix unless we support the chain account `0x0` in fungible.